lib/config-schema.coffee

```diff
@@ -39,6 +39,23 @@ module.exports =
         type: "boolean"
         default: true
 
+      audioclip:
+        title: "Play Audio - Audioclip"
+        description: "Which audio clip played at keystroke."
+        type: "string"
+        default: '../audioclips/gun.wav'
+        enum: [
+          {value: '../audioclips/gun.wav', description: 'Gun'}
+          {value: '../audioclips/typewriter-key-1.wav', description: 'Type Writer'}
+          {value: 'customAudioclip', description: 'PathTo..'}
+        ]
+
+      customAudioclip:
+        title: "Play Audio - Path to Audioclip"
+        description: "Path to audioclip played at keystroke."
+        type: "string"
+        default: 'rocksmash.wav'
```

Comment:
`default` should be empty or removed. Maybe specify what formats are supported on the description.